data/json/construction.json

```diff
@@ -2375,5 +2375,22 @@
         ],
         "pre_terrain" : "t_reinforced_glass",
         "post_terrain" : "t_reinforced_glass_shutter"
+    },{
+        "type" : "construction",
+        "description" : "Build Root Cellar",
+        "category" : "CONSTRUCT",
```

Comment:
Recommend using the `pre_note` to describe what this actually does - as a player I'd be mystified.